.github/workflows/gradle-publish.yml

```diff
@@ -53,12 +50,68 @@ jobs:
           JRELEASER_GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
           JRELEASER_GPG_PASSPHRASE: ${{ secrets.SIGNING_PASSPHRASE }}
 
-      - name: Publish to OSSRH repository
-        run: ./gradlew publish jreleaserFullRelease
+      - name: Publish to Maven Central Portal
+        id: publish
+        run: ./gradlew publish jreleaserFullRelease --info --stacktrace -Djreleaser.verbose=true
         env:
           JRELEASER_MAVENCENTRAL_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           JRELEASER_MAVENCENTRAL_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
           JRELEASER_GPG_PUBLIC_KEY: ${{ secrets.GPG_PUBLIC_KEY }}
           JRELEASER_GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
           JRELEASER_GPG_PASSPHRASE: ${{ secrets.SIGNING_PASSPHRASE }}
           JRELEASER_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Query Central Portal for validation errors
+        if: failure()
+        shell: bash
+        env:
+          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          OSSRH_TOKEN: ${{ secrets.OSSRH_TOKEN }}
+        run: |
+          set -euo pipefail
+          
+          if [[ ! -f build/jreleaser/output.properties ]]; then
+            echo "::error title=File Missing::The file 'build/jreleaser/output.properties' does not exist."
+            exit 1
+          fi
+          
+          DEPLOY_ID=$(grep '^deploymentId=' build/jreleaser/output.properties | cut -d= -f2 || true)
+          if [[ -z "$DEPLOY_ID" ]]; then
+            echo "::error title=Missing Deployment ID::The 'deploymentId' key was not found in 'build/jreleaser/output.properties'."
+            exit 1
+          fi
+          
+          echo "Portal deploymentId: $DEPLOY_ID"
+          
+          AUTH=$(printf "%s:%s" "$OSSRH_USERNAME" "$OSSRH_TOKEN" | base64)
```

Comment:
Maybe, check that variables set before running the script?

Reply:
What do you mean?

Reply:
```
if [ -z "$OSSRH_USERNAME" ]; then
    echo "::error title='OSSRH_USERNAME' is not set."
    exit 1
 fi

if [ -z "$OSSRH_TOKEN" ]; then
    echo "::error title='OSSRH_TOKEN' is not set."
    exit 1
 fi
```
Please ignore, if we are sure that variables exists at that point.